bin/backup.sh

```diff
@@ -44,19 +45,23 @@ if [ "x${TARGET_BUCKET_URL}" == "x" ]; then
   exit 1
 fi
 
-
 # dump database
-if [ "x${MONGODB_DBNAME}" != "x" ]; then
-  MONGODUMP_OPTS="${MONGODUMP_OPTS} -d ${MONGODB_DBNAME}"
-fi
-if [ "x${MONGODB_USERNAME}" != "x" ]; then
-  MONGODUMP_OPTS="${MONGODUMP_OPTS} -u ${MONGODB_USERNAME} -p ${MONGODB_PASSWORD}"
-fi
-if [ "x${MONGODB_AUTHDB}" != "x" ]; then
-  MONGODUMP_OPTS="${MONGODUMP_OPTS} --authenticationDatabase ${MONGODB_AUTHDB}"
+if [ "x${MONGODB_URI}" != "x" ]; then
+  MONGODUMP_OPTS="--uri=${MONGODB_URI} ${MONGODUMP_OPTS}"
```

Comment:
Shouldn't I quote it in case it contains `&`?

Reply:
Tested with a URI containing `&`, but it works as expected.